packages/primitives/button/test/index.ts

```diff
@@ -0,0 +1,56 @@
+import test from 'tape'
+import { getAppPage } from 'foreal'
+
+type TFixtures = 'bare-bones' | 'disabled'
+const getFixture = (filename: TFixtures) => `./fixtures/${filename}.tsx`
```

Comment:
Would it make sense/be possible with an approach closer to the metafiles where we define all props in object form instead of having separate files rendering the component? 

Or maybe just defining the tests in the same file? 

I'm thinking something like this;

```
const tests = [
  {
    fixture: {
      testName: 'Has correct role',
      'aria-label': 'accessibilityLabel'
      onPress: () => {}
    },
    expect: (t, snapshot) => {
      t.equal(a11ySnapBtn.role, 'button', 'button role is `button`')
    }
  },
  {
    fixture: {
      testName: 'Can be disabled',
      'aria-label': 'accessibilityLabel'
      isDisabled: true
      onPress: () => {}
    },
    expect: (t, snapshot) => {
      t.ok(a11ySnapBtn.disabled)
    }
  },
  {
    fixture: {
      testName: 'Can be focused',
      'aria-label': 'accessibilityLabel'
      isDisabled: true
      onPress: () => {}
    },
    expect: (t, snapshot) => {
      t.ok(a11ySnap.focused)
    }
  }
]

```

Reply:
I created some utils for this 👍